src/ngraph/op/quantized_convolution.cpp

```diff
@@ -0,0 +1,196 @@
+//*****************************************************************************
+// Copyright 2018-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "quantized_convolution.hpp"
+#include <numeric>
+#include "ngraph/coordinate_diff.hpp"
+#include "ngraph/op/convolution.hpp"
+#include "ngraph/util.hpp"
+#include "ngraph/validation_util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+op::QuantizedConvolution::QuantizedConvolution(const shared_ptr<Node>& input,
+                                               const shared_ptr<Node>& filters,
+                                               const Strides& window_movement_strides,
+                                               const Strides& window_dilation_strides,
+                                               const CoordinateDiff& padding_below,
+                                               const CoordinateDiff& padding_above,
+                                               const Strides& data_dilation_strides,
+                                               const shared_ptr<Node>& input_scale,
+                                               const shared_ptr<Node>& input_zero_point,
+                                               const shared_ptr<Node>& filter_scale,
+                                               const shared_ptr<Node>& filter_zero_point,
+                                               const shared_ptr<Node>& output_scale,
+                                               const shared_ptr<Node>& output_zero_point,
+                                               const element::Type& output_type,
+                                               const AxisSet& input_axes,
+                                               const AxisSet& filter_axes,
+                                               const AxisSet& output_axes)
```

Comment:
make the axes arguments optional since we dont expect most users to care about these.